### ISSUE TITLE
Cant seem to run pserve 

### ISSUE THREAD

**Issue by ddesmond** [eoyilmaz/stalker_pyramid#2]
hey, hope someone helps.
basicly ivegot stalker and anima running, db connection is fine (postgre). But when starting pserve ivegot the following errors:
`[root@dam01 stalker_pyramid]# /bin/pserve production.ini 
2017-06-26 10:06:49,306 DEBUG [stalker_pyramid][MainThread] LESSOPEN: ||/usr/bin/lesspipe.sh %s
2017-06-26 10:06:49,306 DEBUG [stalker_pyramid][MainThread] SSH_CLIENT: 192.168.5.13 53691 22
2017-06-26 10:06:49,306 DEBUG [stalker_pyramid][MainThread] SELINUX_USE_CURRENT_RANGE: 
2017-06-26 10:06:49,306 DEBUG [stalker_pyramid][MainThread] LOGNAME: root
2017-06-26 10:06:49,306 DEBUG [stalker_pyramid][MainThread] USER: root
2017-06-26 10:06:49,306 DEBUG [stalker_pyramid][MainThread] PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin
2017-06-26 10:06:49,306 DEBUG [stalker_pyramid][MainThread] HOME: /root
2017-06-26 10:06:49,306 DEBUG [stalker_pyramid][MainThread] LANG: en_US.UTF-8
2017-06-26 10:06:49,306 DEBUG [stalker_pyramid][MainThread] TERM: xterm
2017-06-26 10:06:49,307 DEBUG [stalker_pyramid][MainThread] SHELL: /bin/bash
2017-06-26 10:06:49,307 DEBUG [stalker_pyramid][MainThread] SHLVL: 1
2017-06-26 10:06:49,307 DEBUG [stalker_pyramid][MainThread] HISTSIZE: 1000
2017-06-26 10:06:49,307 DEBUG [stalker_pyramid][MainThread] REPO32: /mnt/Z/Projects/
2017-06-26 10:06:49,307 DEBUG [stalker_pyramid][MainThread] XDG_RUNTIME_DIR: /run/user/0
2017-06-26 10:06:49,307 DEBUG [stalker_pyramid][MainThread] SELINUX_ROLE_REQUESTED: 
2017-06-26 10:06:49,307 DEBUG [stalker_pyramid][MainThread] QT_GRAPHICSSYSTEM_CHECKED: 1
2017-06-26 10:06:49,307 DEBUG [stalker_pyramid][MainThread] XDG_SESSION_ID: 2
2017-06-26 10:06:49,307 DEBUG [stalker_pyramid][MainThread] _: /bin/pserve
2017-06-26 10:06:49,307 DEBUG [stalker_pyramid][MainThread] SSH_CONNECTION: 192.168.5.13 53691 192.168.5.53 22
2017-06-26 10:06:49,307 DEBUG [stalker_pyramid][MainThread] SSH_TTY: /dev/pts/0
2017-06-26 10:06:49,307 DEBUG [stalker_pyramid][MainThread] OLDPWD: /opt
2017-06-26 10:06:49,307 DEBUG [stalker_pyramid][MainThread] HOSTNAME: dam01
2017-06-26 10:06:49,308 DEBUG [stalker_pyramid][MainThread] SELINUX_LEVEL_REQUESTED: 
2017-06-26 10:06:49,308 DEBUG [stalker_pyramid][MainThread] HISTCONTROL: ignoredups
2017-06-26 10:06:49,308 DEBUG [stalker_pyramid][MainThread] PWD: /opt/stalker_pyramid
2017-06-26 10:06:49,308 DEBUG [stalker_pyramid][MainThread] MAIL: /var/spool/mail/root
2017-06-26 10:06:49,308 DEBUG [stalker_pyramid][MainThread] LS_COLORS: rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=01;05;37;41:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.Z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.jpg=01;35:*.jpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.axv=01;35:*.anx=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=01;36:*.au=01;36:*.flac=01;36:*.mid=01;36:*.midi=01;36:*.mka=01;36:*.mp3=01;36:*.mpc=01;36:*.ogg=01;36:*.ra=01;36:*.wav=01;36:*.axa=01;36:*.oga=01;36:*.spx=01;36:*.xspf=01;36:
Traceback (most recent call last):
  File "/bin/pserve", line 11, in <module>
    sys.exit(main())
  File "/usr/lib/python2.7/site-packages/pyramid/scripts/pserve.py", line 51, in main
    return command.run()
  File "/usr/lib/python2.7/site-packages/pyramid/scripts/pserve.py", line 316, in run
    global_conf=vars)
  File "/usr/lib/python2.7/site-packages/pyramid/scripts/pserve.py", line 340, in loadapp
    return loadapp(app_spec, name=name, relative_to=relative_to, **kw)
  File "/usr/lib/python2.7/site-packages/PasteDeploy-1.5.2-py2.7.egg/paste/deploy/loadwsgi.py", line 247, in loadapp
    return loadobj(APP, uri, name=name, **kw)
  File "/usr/lib/python2.7/site-packages/PasteDeploy-1.5.2-py2.7.egg/paste/deploy/loadwsgi.py", line 272, in loadobj
    return context.create()
  File "/usr/lib/python2.7/site-packages/PasteDeploy-1.5.2-py2.7.egg/paste/deploy/loadwsgi.py", line 710, in create
    return self.object_type.invoke(self)
  File "/usr/lib/python2.7/site-packages/PasteDeploy-1.5.2-py2.7.egg/paste/deploy/loadwsgi.py", line 146, in invoke
    return fix_call(context.object, context.global_conf, **context.local_conf)
  File "/usr/lib/python2.7/site-packages/PasteDeploy-1.5.2-py2.7.egg/paste/deploy/util.py", line 55, in fix_call
    val = callable(*args, **kw)
  File "/opt/stalker_pyramid/stalker_pyramid/__init__.py", line 126, in main
    config.add_jinja2_renderer('.html')
  File "/usr/lib/python2.7/site-packages/pyramid/config/__init__.py", line 792, in __getattr__
    raise AttributeError(name)
AttributeError: add_jinja2_renderer
`
Ive cheched your master and develop branches, develop throws more error up to this stage (missing files etc,)
What should i do, or better what i did wrong so i can fix it?
thanks
@eoyilmaz

**Comment by eoyilmaz** [comment]
I'll do a commit in two (on vacation now) days that will fix it an kick
start the Angular part.

**Comment by ddesmond** [comment]
Oh!, enjoy the vacation!! dont wory! no hurry!

**Comment by eoyilmaz** [comment]
Returned back from vacation :) I've checked my local development branch and the RESTFul service is working fine. Can you check if you are at the latest commit.

**Comment by eoyilmaz** [comment]
Oh you may need to call the following command to install the ``pyramid_jinja2`` extension::

```
../bin/python setup.py develop
```

If it complains about a missing package, install it manually by using the following command:

```
.../bin/pip install {package_name}
```

or you can use ``easy_install``:

```
.../bin/easy_install {package_name}
```

**Comment by ddesmond** [comment]
will check later and try to do a fresh "install"
Thank you mr. Erkan

**Comment by ddesmond** [comment]
@eoyilmaz 
the error still persists.
Did a fresh docker with centos, updated, installed postgre docker and setup it like in production.ini file.
Pulled stalker, installed, opened py imported fine. pulled anima, installed, opened py import anima run fine.
Pulled stalker pyramid, checkouted on develop, run python setup develop, in virtualenv, erorrs missing files changelog, readme, copied those, then runed again, worked fine, everyno and then i miss some pip install like timecode, pyzt etc.. installed npm in static folder. run init for db complained about missing pip, fixed those, initdb run fine.
pserve thre same error. i installed pyramid_jinja2 but the error persists.

`2017-06-28 16:53:11,480 DEBUG [stalker_pyramid][MainThread] SHLVL: 1
2017-06-28 16:53:11,480 DEBUG [stalker_pyramid][MainThread] SSH_SUDO: ALL=(ALL) ALL
2017-06-28 16:53:11,480 DEBUG [stalker_pyramid][MainThread] HOSTNAME: 8789e280f694
2017-06-28 16:53:11,480 DEBUG [stalker_pyramid][MainThread] SSH_USER_PASSWORD_HASHED: false
2017-06-28 16:53:11,481 DEBUG [stalker_pyramid][MainThread] SSH_USER_ID: 500:500
2017-06-28 16:53:11,481 DEBUG [stalker_pyramid][MainThread] SSH_AUTOSTART_SSHD_BOOTSTRAP: true
2017-06-28 16:53:11,481 DEBUG [stalker_pyramid][MainThread] SSH_USER_SHELL: /bin/bash
2017-06-28 16:53:11,481 DEBUG [stalker_pyramid][MainThread] PWD: /opt/stalker_pyramid/stalker_pyramid
2017-06-28 16:53:11,481 DEBUG [stalker_pyramid][MainThread] SSH_USER_FORCE_SFTP: false
2017-06-28 16:53:11,481 DEBUG [stalker_pyramid][MainThread] SSH_USER: app-admin
2017-06-28 16:53:11,482 DEBUG [stalker_pyramid][MainThread] SSH_USER_PASSWORD: dam
2017-06-28 16:53:11,482 DEBUG [stalker_pyramid][MainThread] HOME: /root
2017-06-28 16:53:11,482 DEBUG [stalker_pyramid][MainThread] SSH_CHROOT_DIRECTORY: %h
2017-06-28 16:53:11,482 DEBUG [stalker_pyramid][MainThread] PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
2017-06-28 16:53:11,482 DEBUG [stalker_pyramid][MainThread] SSH_AUTOSTART_SSHD: true
2017-06-28 16:53:11,483 DEBUG [stalker_pyramid][MainThread] SSH_USER_HOME: /home/%u
2017-06-28 16:53:11,484 DEBUG [stalker_pyramid][MainThread] OLDPWD: /opt/stalker_pyramid
2017-06-28 16:53:11,484 DEBUG [stalker_pyramid][MainThread] _: /usr/bin/pserve
Traceback (most recent call last):
  File "/usr/bin/pserve", line 9, in <module>
    load_entry_point('pyramid==1.9', 'console_scripts', 'pserve')()
  File "/usr/lib/python2.7/site-packages/pyramid/scripts/pserve.py", line 32, in main
    return command.run()
  File "/usr/lib/python2.7/site-packages/pyramid/scripts/pserve.py", line 229, in run
    app = loader.get_wsgi_app(app_name, config_vars)
  File "/usr/lib/python2.7/site-packages/plaster_pastedeploy/__init__.py", line 95, in get_wsgi_app
    global_conf=defaults)
  File "/usr/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 247, in loadapp
    return loadobj(APP, uri, name=name, **kw)
  File "/usr/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 272, in loadobj
    return context.create()
  File "/usr/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 710, in create
    return self.object_type.invoke(self)
  File "/usr/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 146, in invoke
    return fix_call(context.object, context.global_conf, **context.local_conf)
  File "/usr/lib/python2.7/site-packages/paste/deploy/util.py", line 55, in fix_call
    val = callable(*args, **kw)
  File "/opt/stalker_pyramid/stalker_pyramid/stalker_pyramid/__init__.py", line 126, in main
    config.add_jinja2_renderer('.html')
  File "/usr/lib/python2.7/site-packages/pyramid/config/__init__.py", line 879, in __getattr__
    raise AttributeError(name)
AttributeError: add_jinja2_renderer`

**Comment by eoyilmaz** [comment]
Apparently you were using ``production.ini`` whereas I was using the ``development.ini`` and the ``pyramid_jinja2`` include line was missing in ``production.ini``. It is fixed in the latest development version.

**Comment by ddesmond** [comment]
hm, i just pulled, did the same, have the same error.
was browsing the docker file, there is an env var  (?) entry point? 

im certanly missing something in the installing part . 
Is there any guide except the develop branch which i used as a guidineline?
What info you need from me to sort this out? I can write a guide and contribute after i learn how.

let me know.

**Comment by eoyilmaz** [comment]
Can you try running the pserve with the development.ini

**Comment by eoyilmaz** [comment]
And copy & paste the output here.

**Comment by ddesmond** [comment]
here is the output:
`2017-06-29 19:55:11,819 DEBUG [stalker_pyramid][MainThread] HOSTNAME: 8789e280f694
2017-06-29 19:55:11,820 DEBUG [stalker_pyramid][MainThread] SSH_SUDO: ALL=(ALL) ALL
2017-06-29 19:55:11,820 DEBUG [stalker_pyramid][MainThread] LESSOPEN: ||/usr/bin/lesspipe.sh %s
2017-06-29 19:55:11,820 DEBUG [stalker_pyramid][MainThread] SSH_USER_FORCE_SFTP: false
2017-06-29 19:55:11,820 DEBUG [stalker_pyramid][MainThread] SSH_USER: app-admin
2017-06-29 19:55:11,821 DEBUG [stalker_pyramid][MainThread] PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
2017-06-29 19:55:11,821 DEBUG [stalker_pyramid][MainThread] HOME: /root
2017-06-29 19:55:11,821 DEBUG [stalker_pyramid][MainThread] TERM: xterm
2017-06-29 19:55:11,821 DEBUG [stalker_pyramid][MainThread] SSH_AUTHORIZED_KEYS: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAAEADCwqNSPv32cj68Svc/khDrvBp1hynm+0NPp1X+WFIyTyjZ74QT9H5gNY8rngF1vU8VbTRQaa5Igzls77qgDC1WSgwXY8ShvHLUDxa0gnlCnDSwIZjNckYMTw/kn5KpAYP0X/boCZDF2zcxnZ97HuvoDLD4gBz6qnoq+0HA1AGLu7GRMUnhFhzkbLECdjFnwBgrtg17xBehyJYnZd6+X1pysdKN3pmQCG1q3oftVNscBFju0f6G7pQOH69Q3Jh1cTkZDtEyREj9vSkF7wXSwaEOAlljFq/R37HdGMt+yDOn7yZoOz3sf2WdDS+nq81KqtVtilaxLseX52seVZEV7b5piN6YqL25vSmoH2BthSL1eeWHyYxcT66znwqcVq4XTHjujYLUXduPQ35xeaZlk6rakrudfC/wYofYF5qxXUEyJLzyuyDhwJ3crvEf2FptcEpeCxqwGylnLpP/SN36PeF+h1jP3u6so1bH4c7a3j2/hK/zL+UI92Dver2wceUYKpbDfs2sqyB9VzcqOU2suBjYHHhxMPncuHtX1+10gUoAg4fVtSOewRm4ZECoJFykIfBjXG1o7R5LtM4tFFO57+XUB/nYshicuagTuzA+OdJGJutWSGXjw3fsPK2GLUr1L3Fn8BmnHhKyfhccYNnQdtePXEmzKkmUZCyaco+ZRmRhxI1s+r52E+u4RUrRocNHTIeMR3m8fJaTfyX+5pLr2yN2EXxpOWpewICw/Rrve6PvbwCdWYBsvveFujL1epQAgIXxlVc7r93fPJ0gbLfkqjIpe2fXqiPPyAHknggxYatS34hIZDWfe97WQ88Y+pZgvlgAfSYSkElVhtCx8IU1MAYR47ouIH+0vKS8MHshKwuvE7Ky07tPieKJWvWaYnVcWzVShzYbP01mmHk8zdww5A318y2q4pXVXAVqbvY/LIY9KQj50sj0hUYp8J/8ToVTPhEtEAoFClomDKjyUJ1sz5W2859JThUMkNupjBC9bIW3fL2ZbvcbJxiBOMJ1+PPieGx6wRd8uHAriGK4u8lldys5vV9w46X7GdH6IEPZV5Rj6SXdkPbOkGyQEIW2SbAp+mtDT7DwLlBA1tP5ECkLeVXKoHlTGX0ebW5+bIcqUME/khPjF1RRR/Zc0Tmrh2cNRY1c3xhqqNZJ24s80VIIE4RmYnXFvc0IC8rRd9J1+m9DCtqYn3qpfzmohv0tFsTjfAp2ydjwn6h0885sj3qS/0rVvMU0zXlfsYwKEkT72/IcHFK23EFraAu5ktCy/XayzEuvnBH+4coZXGO57B+7s6bYjmvICM2ksQqdqyDgmDVq+oolgd/PAe/wOUf/DvMYMVNSFaVqASLcvIm2417Ww69k= sonja@sonja-SATELLITE-L50-B
2017-06-29 19:55:11,821 DEBUG [stalker_pyramid][MainThread] SHLVL: 2
2017-06-29 19:55:11,821 DEBUG [stalker_pyramid][MainThread] SSH_USER_PASSWORD_HASHED: false
2017-06-29 19:55:11,821 DEBUG [stalker_pyramid][MainThread] SSH_USER_SHELL: /bin/bash
2017-06-29 19:55:11,821 DEBUG [stalker_pyramid][MainThread] SSH_USER_PASSWORD: dam
2017-06-29 19:55:11,822 DEBUG [stalker_pyramid][MainThread] SSH_USER_ID: 500:500
2017-06-29 19:55:11,822 DEBUG [stalker_pyramid][MainThread] SSH_USER_HOME: /home/%u
2017-06-29 19:55:11,822 DEBUG [stalker_pyramid][MainThread] SSH_AUTOSTART_SSHD_BOOTSTRAP: true
2017-06-29 19:55:11,822 DEBUG [stalker_pyramid][MainThread] SSH_AUTOSTART_SSHD: true
2017-06-29 19:55:11,822 DEBUG [stalker_pyramid][MainThread] SSH_CHROOT_DIRECTORY: %h
2017-06-29 19:55:11,822 DEBUG [stalker_pyramid][MainThread] _: /usr/bin/pserve
2017-06-29 19:55:11,822 DEBUG [stalker_pyramid][MainThread] OLDPWD: /opt/stalker_pyramid
2017-06-29 19:55:11,822 DEBUG [stalker_pyramid][MainThread] SSH_INHERIT_ENVIRONMENT: false
2017-06-29 19:55:11,823 DEBUG [stalker_pyramid][MainThread] PWD: /opt/stalker_pyramid/stalker_pyramid
2017-06-29 19:55:11,823 DEBUG [stalker_pyramid][MainThread] LS_COLORS: rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=01;05;37;41:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.Z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.jpg=01;35:*.jpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.axv=01;35:*.anx=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=01;36:*.au=01;36:*.flac=01;36:*.mid=01;36:*.midi=01;36:*.mka=01;36:*.mp3=01;36:*.mpc=01;36:*.ogg=01;36:*.ra=01;36:*.wav=01;36:*.axa=01;36:*.oga=01;36:*.spx=01;36:*.xspf=01;36:
Traceback (most recent call last):
  File "/usr/bin/pserve", line 11, in <module>
    load_entry_point('pyramid', 'console_scripts', 'pserve')()
  File "/usr/lib/python2.7/site-packages/pyramid/scripts/pserve.py", line 32, in main
    return command.run()
  File "/usr/lib/python2.7/site-packages/pyramid/scripts/pserve.py", line 229, in run
    app = loader.get_wsgi_app(app_name, config_vars)
  File "/usr/lib/python2.7/site-packages/plaster_pastedeploy/__init__.py", line 95, in get_wsgi_app
    global_conf=defaults)
  File "/usr/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 247, in loadapp
    return loadobj(APP, uri, name=name, **kw)
  File "/usr/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 272, in loadobj
    return context.create()
  File "/usr/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 710, in create
    return self.object_type.invoke(self)
  File "/usr/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 146, in invoke
    return fix_call(context.object, context.global_conf, **context.local_conf)
  File "/usr/lib/python2.7/site-packages/paste/deploy/util.py", line 55, in fix_call
    val = callable(*args, **kw)
  File "/opt/stalker_pyramid/stalker_pyramid/stalker_pyramid/__init__.py", line 161, in main
    from stalker.config import defaults
ImportError: cannot import name defaults
[root@8789e280f694 stalker_pyramid]#`

**Comment by eoyilmaz** [comment]
Ok this is a new error which is due to the changes in stalker v0.2.21. I'll fix it in a minute an create a new revision.

**Comment by eoyilmaz** [comment]
Ok let's try it again.

**Comment by ddesmond** [comment]
development.ini works
2017-06-29 20:50:55,617 DEBUG [stalker_pyramid][MainThread] /root/Stalker_Storage/{partial_file_path}
Starting server in PID 7588.
Serving on http://0.0.0.0:6543
Will test the web tomorrow and follow up if i find something else to be broken.
the production ini doesnot work yet.
Ty :)

**Comment by eoyilmaz** [comment]
Try to change the database address in the ``production.ini`` or create a database called ``stalker`` which is the name that the ``production.ini`` is referring to (I've accidentally left the development.ini to look at the ``stalker_home`` database, which is specific to my setup).

**Comment by ddesmond** [comment]
have setup both(stalker, stalker_home) so i dont change anything except the docker addres. but now docker crashed and i lost my /opt.
will do another clean setup on virtualbox so i can save my progress and move forward from there.

**Comment by eoyilmaz** [comment]
Ok I'm closing this issue.

**Comment by ddesmond** [comment]
Managed to setup VirtualBox and working fine on localhost:6543 running centos7 latest with python 2.7
im owning you the installer help file. Should i do a pull request?
Also im curious, what the status of the GUI? What is your plan for stalker_pyramid?
What version are you using in production and how its behaving, better, how do the users find it while working? 
What layers of the pipeline are you planing to cover with it (pseudo - what level of abstraction)?

**Comment by eoyilmaz** [comment]
Currently, I'm developing the GUI with **Angular 2/4 + Twitter Bootstrap**. It will take at least 1 or 2 months before we have something meaningful.

The current version that we're using in the studio is based on HTML5 + jQuery and some other JavaScript libraries. The users find it slow and cumbersome at some point. All needs to be addressed in the new release.

We've successfully used the current version (closed source one) in a 4 year production of a full CG feature film (**The Bad Cat** or **Kötü Kedi Şerafettin** in Turkish). We've planned the project and tracked its progress on Stalker Pyramid, we were able to do Dailies on it, see the Resource workload and communicate over it. It was all fun when everyone in the studio wanted to participate.

But for our current projects which are nearly all commercial projects, the users are struggling with it. The asset management part shines. There is no problem with it. They very much like the automation we've given to them.

But the project management part (where we want them to report how much time that they've spend on a particular task and then reschedule the project with that info) is not used as much as I wished it to be.

It is basically because of a mix of the composition of our management team in our studio (lot's of new people doesn't have the required experience) and our lack effort on educating them and the lack of obligation coming from the higher levels of administration.

They like the other fancy stuff where they're able to visually interact with the data (ex. dragging Tasks around to change the task hierarchy, changing the length of the task, assigning people etc.).

The one that I'm developing currently will focus on the user interaction and the total satisfaction of the system. I had to learn a lot of things about UI design and user interaction. I'm confident now, I know the mistakes we did.

For the next stable version, I plan to supply enough UI to the user where they're able to create and manage Project related data (creating Users, Departments, Groups, Projects, Tasks, Assets, Shots, TimeLogs, Notes etc.) fast and easily. Then for the next versions I have big things in my mind that I don't want to disclosure yet.
